src/content/getting-started/architecture/route-agent/_index.en.md

```diff
@@ -1,17 +1,38 @@
-+++
-title =  "Route Agent"
-weight = 30
-+++
+---
+title: "Route Agent"
+---
 
-The Route Agent component runs on every node in each participating cluster. It is responsible for setting up VXLAN tunnels and routing the
-cross cluster traffic from the node to the cluster’s active Gateway Engine which subsequently sends the traffic to the destination cluster.
+The Route Agent component runs on every node in each participating cluster.
+It is responsible for setting up the necessary host network elements on top
+of the existing network plugin.
 
-When running on the same node as the active Gateway Engine, Route Agent creates a VXLAN VTEP interface to which Route Agent instances
-running on the other worker nodes in the local cluster connect by establishing a VXLAN tunnel with the VTEP of the active Gateway Engine
-node. The MTU of the VXLAN tunnel is configured based on the MTU of the default interface on the host minus the VXLAN overhead.
+The Route Agent receives the detected K8s network plugin as part of its
+configuration.
 
-Route Agents use `Endpoint` resources synced from other clusters to configure routes and to program the necessary iptables rules to enable
-full cross-cluster connectivity.
+## kubeproxy-iptables
```

Comment:
The K8s docs seem to consistently refer to it as kube-proxy.

```suggestion
## kube-proxy
```

Reply:
Yes, but there is also kubeproxy IPVS that we don't support, I believe we should keep the distinction I guess kube-proxy iptables

Reply:
done +/- ;)